### PR TITLE
disable functional environment

### PR DESCRIPTION
**Pull request by ebattat**
Fixes:

1. Disable Functional environment
2. Change the Perf CI to run on Tuesday and Friday, (Friday - running OpenShift Assisted installer)

### CODE REVIEW

File: .github/workflows/Nightly_Func_Env_CI.yml [hunk 1]
```diff
@@ -4,13 +4,13 @@ name: Deploy Func Env Nightly CI
...
-    workflows: ["Deploy Operator FUNC Env Weekly CI"]
-    types:
-      - completed
+#  schedule:
+#    - cron: '0 4 * * 1-6' # run Monday-Saturday at 4 AM UTC/ 0 AM EDT
+#  # Run on Sunday after assisted installer and operator completed
+#  workflow_run:
+#    workflows: ["Deploy Operator FUNC Env Weekly CI"]
+#    types:
+#      - completed
```
Comment (RobertKrawitz) [inline .github/workflows/Nightly_Func_Env_CI.yml:13]: Should these (and all of the other functional environment actions) go away altogether?

**Review by RobertKrawitz — COMMENTED** [review]
Only question is whether the func actions should be removed altogether, otherwise looks fine.

File: .github/workflows/Nightly_Func_Env_CI.yml (hunk 1 above)
Comment (ebattat) [reply .github/workflows/Nightly_Func_Env_CI.yml:13]: Yes, we disable the cron job and calling by other workflow

Comment (RobertKrawitz) [reply .github/workflows/Nightly_Func_Env_CI.yml:13]: My question is whether we should remove these files.

Comment (ebattat) [reply .github/workflows/Nightly_Func_Env_CI.yml:13]: We can remove it but I dont see any problem to keep it as is

Comment (RobertKrawitz) [reply .github/workflows/Nightly_Func_Env_CI.yml:13]: Just to clean up unused code.  If we think we may want to bring it back, OK, but I'd like to keep everything tight.  I'm going to approve it; you decide whether you want to do that or not.

**Review by RobertKrawitz — APPROVED** [review]

**Pull request merged**